### ISSUE TITLE
onRowClick not available to CustomRowComponent

### ISSUE THREAD

**Issue by sanju-sfo** [GriddleGriddle/Griddle#287]
I have create a CustomRowComponent and when trying to access the props.onRowClick inside CustomRowComponent it's not available.

``` javascript
<Griddle onRowClick={this.handleClick} useCustomRowComponent={true}  customRowComponent={MyCustomRowComponent} />

var MyCustomRowComponent = React.createClass({
      getDefaultProps: function(){
        return { "data": {} ,
        "onRowClick": null
        };
      }, 
      handleChange: function(e){
        this.props.onRowClick(this, e);
      }
}
);
```

This where it fails with error Uncaught TypeError: this.props.onRowClick is not a function

**Comment by yordis** [comment]
@sanju-sfo  https://github.com/GriddleGriddle/Griddle/issues/198

**Comment by ryanlanciaux** [comment]
Hi there and sorry about the delay! We've recently released Griddle 1.0 and have added quite a bit more options around customization. Please [see the latest customization documentation for more information](http://griddlegriddle.github.io/Griddle/docs/customization/). Closing this one for now but please feel free to open a new issue if any problems are encountered. :+1:
